### PR TITLE
fix broken link in references

### PR DESCRIPTION
**Pull request by retraigo**
Changed `STD_VERSION` to `CLI_VERSION`

### CODE REVIEW

File: references.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This chapter contains links to reference manuals and sheets, including:
 
-- [Deno Standard APIs](https://deno.land/api@v$STD_VERSION)
+- [Deno Standard APIs](https://deno.land/api@$CLI_VERSION)
```
Comment (kt3k) [inline references.md:5]:
```suggestion
- [Deno Standard APIs](https://deno.land/api@v$CLI_VERSION)
```

**Review by kt3k — APPROVED** [review]
LGTM

File: references.md (hunk 1 above)
Comment (retraigo) [reply references.md]: That actually broke the link again (since `$CLI_VERSION` is already prefixed with a `v`)

Comment (kt3k) [reply references.md]: Oops! Sorry I didn't realize it

Comment (kt3k) [reply references.md]: There were many same mistakes https://github.com/denoland/manual/pull/439

**Pull request merged**